release_docs/RELEASE_PROCESS.md

```diff
@@ -161,3 +161,19 @@ For more information on the HDF5 versioning and backward and forward compatibili
 ### 11. Conduct Release Retrospective (Release Manager)
 1. Schedule time and solicit comments from retrospective
 2. Identify issues and document them
+
+[u1]: https://github.com/HDFGroup/hdf5/blob/develop/release_docs/RELEASE.txt
```

Comment:
Same comment here about using 1.14 versions of the links rather than develop

Reply:
Yes